osquery/core/windows/utils.cpp

```diff
@@ -20,6 +22,15 @@
 
 namespace osquery {
 
+LONGLONG filetimeToUnixtime(FILETIME ft) {
```

Comment:
Are you using this function somewhere else now? Or just moving it for good measure? If so, now might be a good time to turn the input parameter into a `const FILETIME&` since it seems to be used as a struct. A constant reference prevents the unneeded copy.